sdk/servicebus/azure-servicebus/README.md

```diff
@@ -187,14 +192,37 @@ with ServiceBusClient.from_connection_string(connstr) as client:
     with client.get_subscription_session_receiver(topic_name, subscription_name, session_id) as receiver:
         for msg in receiver:
             print(str(msg))
+```
+
+### Settle a message after receipt
+
+When receiving from a queue, you have multiple actions you can take on the messages you receive.  
+
+> **NOTE**: You can only settle `ReceivedMessage` objects having been returned from `ServiceBusReceiver.receive()` and `ServiceBusReceive.__iter__()` in `ReceiveSettleMode.PeekLock` mode
+> (this is the default).  `ReceiveSettleMode.ReceiveAndDelete` mode removes the message from the queue on receipt as if `complete()` had been called on it.  `PeekMessage` messages
```

Comment:
I'd rather not mentioning "as if `complete()` had been called on it." as this may bring some ambiguity, like users might be wondering does the SDK call "complete" for us.

Reply:
I had a similar concern but was hoping that drawing a parallel behind concepts would be more clear.  If you saw this risk though as well I'll yank that segment, we can always readd if that seems to be the "missing gap" in folks understanding this, I _think_ it's plenty sufficient even without.